### PR TITLE
feat: minor content updates for index and oathkeeper

### PR DESCRIPTION
**Pull request by jfcurran**

### CODE REVIEW

**Review by aeneasr — APPROVED** [review]

**Comment by aeneasr** [comment]
@tacurran please review

File: src/components/footer.tsx [hunk 1]
```diff
@@ -47,23 +47,23 @@ interface MenuItem {
...
   {
     title: 'Products',
-    href: '/products',
+    href: '/',
     items: [
       {
-        title: 'OAuth 2.0',
-        href: '/hydra'
+        title: 'User Management',
+        href: '/kratos'
```
Comment (zepatrik) [inline src/components/footer.tsx:54]: can we maybe keep the order here to see what actually changed? it is very hard to tell right now

File: src/components/thin-project.module.css [hunk 2]
```diff
@@ -39,3 +39,7 @@
   flex-direction: row-reverse;
   justify-content: flex-end;
 }
+
+.capital  {
+  text-transform: capitalize;
+}
```
Comment (zepatrik) [inline src/components/thin-project.module.css:45]:
```suggestion
}

```

File: src/config.tsx [hunk 3]
```diff
@@ -48,9 +48,9 @@ export const projects: Project[] = [
   {
     id: 'oathkeeper',
     title: `${brandPrefix}Oathkeeper`,
-    descriptiveTitle: 'Identity and Access Proxy',
+    descriptiveTitle: 'Identity and Access Proxy (IAP)',
     description:
-      'Identity and Access Proxy (IAP). Authenticate and authorize all traffic, using Zero Trust / BeyondCorp as open source.',
+      'Configure how traffic interacts with your application. Authenticate and authorize all traffic, using Zero Trust / BeyondCorp as open source.',
```
Comment (zepatrik) [inline src/config.tsx:53]:
```suggestion
      'Regulate what traffic interacts with your application. Authenticate and authorize all traffic, using Zero Trust / BeyondCorp as open source.',
```
The wording here seems a bit unprecise and inaccurate, this is just one idea.

File: src/pages/index.tsx [hunk 4]
```diff
@@ -42,94 +41,41 @@ const IndexPage = () => (
...
-          visual: 'hydra',
-          title: 'OAuth 2.0 and OpenID Connect',
-          href: '/hydra',
+          visual: 'opensource',
+          title: 'Powered by Open Source',
+          href: 'https://github.com/ory',
+          openInNewWindow: true,
           content:
-            'Keep threats out and easily verify who is a customer, and customize sign in flows that fit you and your customers.'
+            'You can rely on open source software that is built together in a world-wide community of developers. Peer reviewed, vetted by expert developers - tried and tested.'
```
Comment (zepatrik) [inline src/pages/index.tsx:49]: I like the imperative more, also the other texts are imperative
```suggestion
            'Rely on open source software that is built together in a world-wide community of developers. Peer reviewed, vetted by expert developers - tried and tested.'
```

File: src/pages/index.tsx [hunk 5]
```diff
@@ -42,94 +41,41 @@ const IndexPage = () => (
...
-          visual: 'keto',
-          title: 'Authorization',
-          href: '/keto',
+          visual: 'integration',
+          title: 'Easy integration',
+          href: 'https://github.com/ory/sdk',
+          openInNewWindow: true,
           content:
-            'Personalize experiences using customer specific access rules, define roles, and give customers individual access to your service.'
+            'Work with what you’re doing. Easy ways to make tools, software, platforms and coding languages used by your business already work with Ory.'
```
Comment (zepatrik) [inline src/pages/index.tsx:57]: The second sentence is quite cumbersome, it took me 5 reads to get it. Maybe change to a different verb than `make .................................................................................................... work`.
```suggestion
            'Work with what you’re doing. Ory enables integrations with tools, software, platforms and coding languages used by your business already.'
```

Comment (zepatrik) [reply src/pages/index.tsx:57]: Maybe "integreations"? :joy:

File: src/pages/index.tsx [hunk 6]
```diff
@@ -42,94 +41,41 @@ const IndexPage = () => (
...
         },
         {
-          visual: 'kratos',
-          title: 'User management',
-          href: '/kratos',
+          visual: 'openstandards',
+          title: 'Rely on open standards',
```
Comment (zepatrik) [inline src/pages/index.tsx:61]: I like it, just some other idea:
```suggestion
          title: 'Build on open standards',
```
"Rely" sounds like you are depending on someone else, while "build" makes you the leader.

**Review by zepatrik — CHANGES_REQUESTED** [review]
Nice, like the way this is going :+1:

**Comment by aeneasr** [comment]
What's the status of this?

**Pull request closed without merging**